src/engine/client/key_binding.cpp

```diff
@@ -269,6 +269,17 @@ void SetBinding(Key key, int team, std::string binding)
 
 // -ve team no. = don't return the default binding
```

Comment:
That can we push the boolean to the caller? That negative argument thing is pretty ew.